core/test/XiosCalendar_test.cpp

```diff
@@ -48,6 +48,7 @@ MPI_TEST_CASE("TestXiosInitialization", 2)
     REQUIRE(start == xios_handler.getCalendarStart());
     REQUIRE(start.format() == "2023-03-17T17:11:00Z");
     // Timestep
+    REQUIRE(xios_handler.getCalendarTimestep().seconds() == doctest::Approx(3600.0)); // Default
     Duration timestep("P0-0T01:30:00");
     REQUIRE(timestep.seconds() == doctest::Approx(5400.0));
```

Comment:
I know that testing floating point values can be imprecise, but I'd suggest that if we don't get back the exact integer values of 3600 and 5400, that should also be a test failure.

Reply:
Good point - fixed in 2eec43ffe8ec19a2918ee5645ca9b9225786956a.